docs/querying/functions.md

```diff
@@ -434,3 +434,45 @@ over time and return an instant vector with per-series aggregation results:
 
 Note that all values in the specified interval have the same weight in the
 aggregation even if the values are not equally spaced throughout the interval.
+
+## Trigonometric Functions
+
+- `acos(v instant-vector)`: calculates the arccosine, in radians, of all elements in `v`. Special cases are:
+    - `acos(x) = NaN if x < -1 or x > 1`
+- `asin(v instant-vector)`: calculates the arcsine, in radians, of all elements in `v`. Special cases are:
+    - `asin(±0) = ±0`
+- `atan(v instant-vector)`: calculates the arctangent, in radians, of all elements in `v`. Special cases are:
+    - `atan(±0) = ±0`
+    - `atan(±Inf) = ±Pi/2`
+- `atan2(y, x instant-vector)`: calculates the arctangent of `y`/`x` for all elements, in radians, using the signs of the two to determine the quadrant of the result ([from the package](https://pkg.go.dev/math#Atan2)). Special cases are:
```

Comment:
The best explanation I could come up with is mostly lifted from the `math` docs, so I included a link to it.